### PR TITLE
SSPCommonTemplatesModificationReverted updates

### PR DESCRIPTION
**Pull request by apinnick**
Signed-off-by: Avital Pinnick <apinnick@redhat.com>

### CODE REVIEW

File: docs/runbooks/SSPCommonTemplatesModificationReverted.md [hunk 1]
```diff
@@ -1,25 +1,38 @@
...
-	- `kubectl -n $NAMESPACE logs --tail=-1 -l control-plane=ssp-operator`
-	- Look for lines similar to this to find the names of the restored Common Templates:
-	  `Changes reverted in common template: <template name>`
+1. Export the `NAMESPACE` environment variable:
+  ```bash
+  $ export NAMESPACE="$(kubectl get deployment -A | grep ssp-operator | awk '{print $1}')"
+  ```
+2. Check the `ssp-operator` logs for templates with reverted changes:
+  ```bash
+  $ kubectl -n $NAMESPACE logs --tail=-1 -l control-plane=ssp-operator
```
Comment (apinnick) [inline docs/runbooks/SSPCommonTemplatesModificationReverted.md:22]: @sradco 
Would it make sense to add a `| grep 'Changes reverted'` to this command?

**Comment by apinnick** [comment]
@sradco Ready for review

File: docs/runbooks/SSPCommonTemplatesModificationReverted.md (hunk 1 above)
Comment (sradco) [reply docs/runbooks/SSPCommonTemplatesModificationReverted.md:22]: @borod108 please see Avital's question

Comment (sradco) [reply docs/runbooks/SSPCommonTemplatesModificationReverted.md:22]: @borod108 ping

Comment (borod108) [reply docs/runbooks/SSPCommonTemplatesModificationReverted.md:22]: yes, good idea!

Comment (borod108) [reply docs/runbooks/SSPCommonTemplatesModificationReverted.md:22]: maybe we can add "-C 3" to the command tough to see context.

**Comment by apinnick** [comment]
@sradco Ready for review

**Pull request merged**